client/packages/common/src/ui/components/navigation/ListOptions/ListOptions.tsx

```diff
@@ -56,6 +56,7 @@ export const ListOptions = ({
         maxHeight: height * 0.6,
         overflow: 'auto',
         scrollBehavior: 'smooth',
+        position: 'fixed',
```

Comment:
Not 100% sold on this, so keep an eye out for problems, particularly on different screen sizes/devices

Reply:
Didn't see any issue on my andorid.

Reply:
I found what was bothering me -- the text for the items doesn't wrap when in `fixed`, so it can sometimes overlap the rest of the form. Turns out that "fixed" removes the block from the main document flow, so can cause all kinds of other problems. Have removed the "fixed" for now and updated the `maxHeight` to be a bit more useful, and make sure it scrolls.

6d066b42ae